birdie_snapshots/snap_dummy_src_basic_panic_gleam.accepted

```diff
@@ -5,7 +5,7 @@ title: ./snap_dummy/src/basic_panic.gleam
 Path: ./snap_dummy/src/basic_panic.gleam
 
 Location Identifier: this_code_panics
-Rule: NoPanic
+Rule: no_panic
```

Comment:
The rule name resembles the path to the file a lot more, which I think is more useful and more searchable in the configuration.

Reply:
Completely agree